### PR TITLE
chore(CPlusPlus): add largest and smallest number in an array

### PR DESCRIPTION
**Pull request by Uma-95**
**PR Checklist:**
- [x] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [x] My addition is on refer to the language README.md file
- [x] My addition does not have a spelling problem
- [x] My submission has a proper and user-friendly description of the algorithm
- [x] My submission has the time complexity of the algorithm
- [x] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [x] Other, please describe:

**Briefly describe the changes in this PR**
This is a simple program that is used to find the smallest and largest numbers in an array using sort function of C++ STL library.

**Other information:**
For sort function:
Time Complexity: O(N log N)
Auxiliary Space: O(1)

### CODE REVIEW

**Comment by Uma-95** [comment]
@RK-Shandilya I'm contributing to a repository for the first time. Kindly review my PR. Looking forward to your feedback.

**Comment by Uma-95** [comment]
This is my first time contributing to a repository which is why I chose to add a simple program to this repository.
Kindly review my PR and I'm looking forward to some feedback.
@Arsenic-ATG @UG-SEP @aayushjain7 @Ashborn-SM @Ashad001

File: algorithms/CPlusPlus/Arrays/Largest and Smallest Number in an array.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+#include<bits/stdc++.h>
+using namespace std;
+
+//simple approach:
+//sort the array in ascending order.
+//the first element would be the smallest and the last elemnet would be the largest
```
Comment (Arsenic-ATG) [inline algorithms/CPlusPlus/Arrays/Largest and Smallest Number in an array.cpp:7]: I don't know how it actually went under the codespell's radar, but spelling of element is wrong here.

```suggestion
//the first element would be the smallest and the last element would be the largest
```

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]
Apart from the review suggest in review comments, it is also mandatory to update corresponding index README.md file ( `algorithms/CPlusPlus/README.md` in your case ) with an entry of your program in it.

look at a successfully merged pull request to see how it's done.

**Review by Arsenic-ATG — APPROVED** [review]

**Pull request merged**